.ci/packaging.groovy

```diff
@@ -149,75 +159,174 @@ pipeline {
             }
           }
         }
+        stage('Run E2E Tests for Packages'){
+          agent { label 'ubuntu && immutable' }
```

Comment:
```suggestion
          agent { label 'ubuntu-18 && immutable' }
```

Reply:
Can we apply this to master and then backport?